client/asset/eth/nodeclient_harness_test.go

```diff
@@ -129,6 +122,21 @@ var (
 	//
 	// TODO: Make this also work for token tests.
 	isTestnet = false
```

Comment:
I'm running like `TARGS=rpc\ testnet go test --tags lgpl,harness` is this correct? It works but I'm not sure if the intended way to set `TARGS`

Reply:
I do `TARGS="rpc tesnet" go test ...`, but yeah.

Reply:
Flags work in tests, simpler:

```go
func TestMain(m *testing.M) {
	dexeth.MaybeReadSimnetAddrs()

	flag.BoolVar(&isTestnet, "testnet", false, "use testnet")
	flag.BoolVar(&useRPC, "rpc", false, "use RPC")
	flag.Parse()

	if isTestnet {
		if err := useTestnet(); err != nil {
			fmt.Fprintf(os.Stderr, "error loading testnet: %v", err)
			os.Exit(1)
		}
	}
```

test:

```go
func TestFlags(t *testing.T) {
	t.Logf("isTestnet? %v", isTestnet)
	t.Logf("useRPC? %v", useRPC)
}
```

```sh
$  go test -v -run TestFlags
=== RUN   TestFlags
    wsconn_test.go:40: isTestnet? false
    wsconn_test.go:41: useRPC? false
--- PASS: TestFlags (0.00s)
$  go test -v -run TestFlags -testnet
=== RUN   TestFlags
    wsconn_test.go:40: isTestnet? true
    wsconn_test.go:41: useRPC? false
--- PASS: TestFlags (0.00s)
$  go test -v -run TestFlags -testnet -rpc
=== RUN   TestFlags
    wsconn_test.go:40: isTestnet? true
    wsconn_test.go:41: useRPC? true
--- PASS: TestFlags (0.00s)
PASS
```

Reply:
When and where you call Parse has gottas.  Maybe that was it